react-native/src/blueDarkTheme.ts

```diff
@@ -11,19 +11,30 @@ import { blue, red, darkBlack, black, lightBlue } from '@pxblue/colors';
 import { fontConfig } from './shared';
 import Color from 'color';
 
-export const blueDarkTheme: typeof DefaultTheme = {
+declare global {
```

Comment:
Do you want to add a comment saying that this isn't working the way that we expect it to yet?

Reply:
And should we also add these to the light theme? I could see a textSecondary being useful in the future perhaps.